docs/app/pages/css/sections/labels/labels/labels.component.html

```diff
@@ -1,58 +1,126 @@
 <p>General Labels</p>
-<span class="label label-primary m-r-sm">Label Primary</span>
-<span class="label label-accent m-r-sm">Label Accent</span>
-<span class="label label-grey4 m-r-sm">Label Grey4</span>
+<span class="label label-primary m-r-sm">Featured</span>
+<span class="label label-accent m-r-sm">Recently Added</span>
+<span class="label label-grey4 m-r-sm">On Hold</span>
 
 <p class="m-t-md">Outline Labels</p>
-<span class="label label-outline-alternate2 m-r-sm">Outline Label Alternative2</span>
-<span class="label label-outline-alternate3 m-r-sm">Outline Label Alternative3</span>
-<span class="label label-outline-grey4">Outline Label Grey4</span>
+<span class="label label-outline-vibrant2 m-r-sm">Contact</span>
+<span class="label label-outline-vibrant1 m-r-sm">Company</span>
+<span class="label label-outline-alternate1">Location</span>
 
 <p class="m-t-md">Status Labels</p>
-<span class="label label-ok m-r-sm">Label OK</span>
-<span class="label label-warning m-r-sm">Label Warning</span>
-<span class="label label-critical">Label Critical</span>
+<span class="label label-ok m-r-sm">Completed</span>
+<span class="label label-warning m-r-sm">Cancelled</span>
+<span class="label label-critical">Failed</span>
 
 <hr>
 
 <p>Use <code>span.label</code> to create a label. UX Aspects offers a wide range of label &amp; outline label colors.
 Set the color of the label using one of the following classes:</p>
 
-<p>General Labels:</p>
-
-<p><code>.label-primary</code> or
-<code>.label-accent</code> or
-<code>.label-alternate1</code> or
-<code>.label-alternate2</code> or
-<code>.label-alternate3</code> or
-<code>.label-vibrant1</code> or
-<code>.label-vibrant2</code> or
-<code>.label-grey1</code> or
-<code>.label-grey2</code> or
-<code>.label-grey3</code> or
-<code>.label-grey4</code> or
-<code>.label-grey5</code></p>
-
-<p>Outline Labels:</p>
-
-<p><code>.label-outline-primary</code> or
-<code>.label-outline-accent</code> or
-<code>.label-outline-alternate1</code> or
-<code>.label-outline-alternate2</code> or
-<code>.label-outline-alternate3</code> or
-<code>.label-outline-vibrant1</code> or
-<code>.label-outline-vibrant2</code> or
-<code>.label-outline-grey1</code> or
-<code>.label-outline-grey2</code> or
-<code>.label-outline-grey3</code> or
-<code>.label-outline-grey4</code> or
-<code>.label-outline-grey5</code></p>
-
-<p>Status Labels:</p>
-
-<p><code>.label-ok</code> or
-<code>.label-warning</code> or
-<code>.label-critical</code></p>
+<div class="demo-attributes table-responsive">
+    <table class="table">
+        <tbody>
+            <tr>
+                <th>General Labels</th>
+                <th></th>
+                <th>Outline Labels</th>
+                <th></th>
```

Comment:
I think this would be better as two tables in column layout. If you use the col-* classes, you can get them to appear vertically for smaller screens.

See how it looks like this:

````
<div class="row">
  <div class="col-md-6 col-sm-12">
    <!-- General labels -->
  </div>
  <div class="col-md-6 col-sm-12">
    <!-- Outline labels -->
  </div>
</div>
````

See how that looks, it might be good with a small gap between the columns too.